tide_core.install

```diff
@@ -421,3 +421,23 @@ function tide_core_update_10015() {
   // Save the configuration.
   $config->save();
 }
+
+/**
+ * Add code deployed tag to environment.
+ */
+function tide_core_update_10016() {
+  $env = getenv('LAGOON_ENVIRONMENT_TYPE') ?: 'development';
+  $moduleHandler = \Drupal::service('module_installer');
+  if (!$moduleHandler->moduleExists('environment_indicator')) {
+    $moduleHandler->install(['environment_indicator']);
+  }
+
+  if ($env !== 'production') {
```

Comment:
hey @sharmasahil 
I don't think we have a config split for user roles, so checking `$env !== 'production'` might not be necessary.

Reply:
hey @sharmasahil , You want the update hook to run only in non-prod environments, right? But in reality, this setting will always be exported. e.g. when we run an automation upgrade, it’ll still get deployed to production. We don’t maintain 2 separate databases to keep prod and non-prod configs, and we don't mange roles in different environments . Plus, update hooks only run once, so your setting would still get overwritten by the PROD config.

Reply:
Why do we want it to run in non prod only @sharmasahil / @vincent-gao? This seems useful for production as well?

Reply:
Ah, I just read the AC of the JIRA ticket. Carry on.